addons/im_livechat/static/tests/embed/chat_window_tests.js

```diff
@@ -17,19 +19,24 @@ QUnit.test("do not save fold state of temporary live chats", async (assert) => {
     await start({
         mockRPC(route, args) {
             if (route === "/discuss/channel/fold") {
-                assert.step(`fold - ${args.state}`);
+                step(`fold - ${args.state}`);
             }
         },
     });
     await click(".o-livechat-LivechatButton");
+    await contains(".o-mail-Message", { text: "Hello, how may I help you?" });
+    await assertSteps([]);
     await insertText(".o-mail-Composer-input", "Hello");
-    triggerHotkey("Enter");
+    await triggerHotkey("Enter");
     await contains(".o-mail-Message", { text: "Hello" });
+    await assertSteps(["fold - open"]);
```

Comment:
Note: extra RPC because there was a condition on store.self before init messaging that was (incorrectly) failing due to missing data

Reply:
We might want to remove that RPC at a later time:
- other PR to merge RPC together,
- or if possible better initial fold state so that we don't have to call that at all.